todo_app/tests/test_integrations.py

```diff
@@ -12,293 +14,41 @@ def client():
     with test_app.test_client() as client:
         yield client
 
-class mock_get_lists_response:
-    @staticmethod
-    def json():
-        null=None; false=False
-        listsjson = ([
-        {
-            "id":"fake_trello_todo_listid",
-            "name":"To Do",
-            "closed":false,
-            "pos":65535,
-            "softLimit":null,
-            "idBoard":"fake_trello_boardid",
-            "subscribed":false
+test_todos = [
+    {
+        "_id":"ObjectID(609e3da1407636a2a8b17a58)", 
+        "title": "TestItem1", 
+        "Lastmodifieddate" : "2021-05-13T09:06:41.902+00:00"},
```

Comment:
This is really close - there are two issues; one is that your app is setting up the DB before you mock it (which I'll suggest a solution to elsewhere) and the second is that "lastmodifieddate" here should be all lower case (no leading capital L)